app/experimenter/nimbus-ui/README.md

```diff
@@ -28,6 +28,51 @@ This project uses [Storybook](https://storybook.js.org/) to visually show each c
 
 In local development, `yarn storybook` will start a Storybook server at <http://localhost:3001> with hot module replacement to reflect live changes. We plan to push Storybook builds from pull requests and commits to a GH pages URL in the very near future.
 
+## Writing styles
+
+This app uses [Bootstrap v4.5](https://getbootstrap.com/) for styles. There are two ways for you to use it within code:
+
+- [Bootstrap](https://getbootstrap.com/), in other words all the [global styles and classes](https://getbootstrap.com/docs/4.5/layout/overview/) provided by Bootstrap, is available for use without any additional effort. Just write the tags and/or apply the classes you desire.
+  - Keep in mind that Bootstrap provides a variety of [utility classes](https://getbootstrap.com/docs/4.5/utilities/borders/) for one-off styles that aren't covered by a component class.
+- [`react-bootstrap`](https://react-bootstrap.github.io/) provides us with Bootstrap classes as React components.
+  - [Layout components](https://react-bootstrap.github.io/layout/grid/) give access to Bootstrap's grid system for responsively laying out and aligning content. Example:
+    ```tsx
+    <Container fluid="md">
+      <Row>
+        <Col>1 of 1</Col>
+      </Row>
+    </Container>
+    ```
+  - A [UI component](https://react-bootstrap.github.io/components/alerts/) encapsulates a category of UI, such as a button or alert banner, and tailors how it looks through the props. Example:
+    ```tsx
+    <Alert variant="danger">
+      This is a Danger alert!
+    </Alert>
+    ```
+
+**What about the JavaScript plugins?** While Bootstrap does provide optional JavaScript plugins, they are written in jQuery, which we are decidedly not using in this codebase. You are free to build your own equivelenet in vanilla JavaScript or as a React component, or introduce a new package (within reason).
```

Comment:
~equivelenet~ equivalent

Also do we want to "allow" vanilla JS? I think that should be avoided and don't think we'll need it for this project, I think everything we need to do can just be done inside the React components.

Reply:
I'm not sure what you mean by allow -- by vanilla JavaScript I just meant any JS without a backing library. Did you have something else in mind?